README.md

```diff
@@ -15,9 +15,17 @@ The master node is also running the main rancher container:
 https://172.19.250.22:8443/
 ```
 
-The credentials to access Rancher can be found in the standard MVP Studio secrets bucket (`passwords.mvpstudio.org`
-bucket on GCE). Once in Rancher you can click the `Kubeconfig file` link to get a `~/.kube/config` file. You can also
-find the ssh credentials for the nodes in that bucket in the `continu-k8.txt` file.
+Authentication is via github so log in via github using any github user that is part of the MVPStudio organization. Once
+you've logged in an administrator will add you to our cluster and then whatever projects you are working on. Once in
```

Comment:
Perhaps include that people will need to be on a VPN to log into the cluster, and provide a link to the WireGuard instructions.

Reply:
Line 9 starts with "In order to access the cluster you must first VPN into the Continu cluster. See the
[VPN_README.md](./docs/VPN_README.md) in this directory for details. Assuming you have a running VPN connection then you can connect to the master node."

There is a bug as the VPN docs aren't in this directory so I'll fix that.